cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -232,6 +232,8 @@ type NodeConfiguration struct {
 	// NodeName is the name of the node to join the cluster. Defaults
 	// to the name of the host.
 	NodeName string `json:"nodeName"`
+	// KubeProxy holds configuration for the k8s service proxy.
+	KubeProxy KubeProxy
```

Comment:
This won't work in practice and doesn't belong here. Please remove.

Reply:
remove it, and just gave `warning` when `kubeadm join` node unconditionly